packages/upload/package.json

```diff
@@ -26,7 +26,7 @@
     "lodash.map": "4.6.0",
     "react": "^16.8.0",
     "react-dom": "^16.8.0",
-    "react-dropzone": "^7.0.1",
+    "react-dropzone": "^10.0.0 ||^9.0.0 ||^8.0.0 ||^7.0.1",
```

Comment:
Did `^>=7.0.1` not work?

Reply:
I didn't realize I could use that syntax to be honest

Reply:
Yeah something like: https://github.com/Availity/availity-react/blob/f9d07a08e0f4f2b5aab7762650aef29280345482/packages/authorize/package.json#L35